### PR TITLE
fix: close modal

### LINKED ISSUE #72: Use page heading with cross to close page and go back to tabs
Template Selector: `.styles .page-heading--icon-left`

Behaviour:
- If you're on a CMS page, go to the breadcrumb level of the tabs
- If you're on a client-side only page, go to / (or suggest something better?)

https://trello.com/c/1pKAas0e/164-use-page-heading-with-cross-to-close-page-and-go-back-to-tabs

### PR DESCRIPTION
**Pull request by schalkneethling**
Enable the first iteration of the close modal functionality.
This also required an update to all modal so that they only pass their title and not set their own page title.

fix #72

### CODE REVIEW

File: src/js/components/pages.js [hunk 1]
```diff
@@ -218,7 +219,11 @@ export class PersonPage extends Page {
   }
 
   initContacts(content) {
-    return content.person_contacts.map((details) => new Contact(details));
+    if (content && content.person_contacts) {
+      return content.person_contacts.map((details) => new Contact(details));
+    } else {
+      return [];
+    }
```
Comment (schalkneethling) [inline src/js/components/pages.js:226]: This threw an error the broke the page if `content` is undefined.

**Comment by jbothma** [comment]
I thikn you missed these pages, but even if you implement it here, I think a problem with this approach is if you land on a modal page, there's nothing behind it?

Screenshot_2020-12-10_11-57-44

That's better demonstrated with landing on a service page and clicking close

Screenshot_2020-12-10_11-58-55

Also, this doesn't update the address bar by pushing to history or otherwise, which can break some things like sharing the current page.

For that reason, I think treating "close" as "opening /" is a better MVP unless there's an easy solution to all these issues. (Remember to still add it to the email flow pages)

**Comment by jbothma** [comment]
While I loudly and regularly criticise reddit's mobile web interface and its antipatterns, this aspect of it does actually work and is an anchor tag that takes you to something higher up in the hierachy, so I think taking you to / or the tab in the breadcrumb trail is actually alright for now

Screenshot_2020-12-10_12-05-55

**Comment by schalkneethling** [comment]
> I thikn you missed these pages, but even if you implement it here, I think a problem with this approach is if you land on a modal page, there's nothing behind it?

Yup, that is problematic. I did notice that. I guess as it can be used in all kinds of scenarios, one can possibly land on a modal page without having visited either services or mymuni first

**Pull request merged**